enterprise-suite/console-api/prometheus.yml

```diff
@@ -140,6 +140,8 @@ scrape_configs:
         action: keep
       - action: labelmap
         regex: __meta_kubernetes_service_label_(.+)
+      - source_labels: [__meta_kubernetes_namespace]
+        target_label: namespace
```

Comment:
this is the wrong way to debug the UI test. this is a bad tag, let's figure out what's happening with the UI tests without reverting this fix